README.md

```diff
@@ -1,7 +1,7 @@
 <img src="https://dadi.tech/assets/products/dadi-web-full.png" alt="DADI Web" height="65"/>
 
 [![npm (scoped)](https://img.shields.io/npm/v/@dadi/web.svg?maxAge=10800&style=flat-square)](https://www.npmjs.com/package/@dadi/web)
-[![coverage](https://img.shields.io/badge/coverage-81%25-yellow.svg?style=flat?style=flat-square)](https://github.com/dadi/web)
+[![coverage](https://img.shields.io/badge/coverage-80%2525-yellow.svg?style=flat?style=flat-square)](https://github.com/dadi/web)
```

Comment:
this looks a bit odd, Eduardo had an issue with this earlier too. Modify scripts/coverage.js to not do the badgeUrl.replace step...